eclcli/mlb/v1/mlb_certificate.py

```diff
@@ -293,3 +306,14 @@ def _set_statuses_for_display(data):
 
     ssl_key = data.ssl_key["status"]
     setattr(data, 'ssl_key_status', ssl_key)
+
+
+def _set_infomations_for_display(data):
+    ca_cert = data.ca_cert["info"]
+    setattr(data, 'ca_cert_info', json.dumps(ca_cert, indent=2))
+
+    ssl_cert = data.ssl_cert["info"]
+    setattr(data, 'ssl_cert_info', json.dumps(ssl_cert, indent=2))
+
+    ssl_key = data.ssl_key["info"]
+    setattr(data, 'ssl_key_info', json.dumps(ssl_key, indent=2))
```

Comment:
```suggestion
    ca_cert = data.ca_cert["info"]
    setattr(data, 'ca_cert_info', json.dumps(ca_cert, indent=2, ensure_ascii=False))

    ssl_cert = data.ssl_cert["info"]
    setattr(data, 'ssl_cert_info', json.dumps(ssl_cert, indent=2, ensure_ascii=False))

    ssl_key = data.ssl_key["info"]
    setattr(data, 'ssl_key_info', json.dumps(ssl_key, indent=2, ensure_ascii=False))
```
json.dump関数はデフォルトでは日本語（非ASCII文字）を `\uXXXX` 形式に Unicodeエスケープしてしまいます。
このため `ensure_ascii=False` を指定する必要があります。
この問題については過去に [Slackで議論](https://bespin-dev.slack.com/archives/C01UJ0B686N/p1734317289763869?thread_ts=1731309291.888819&cid=C01UJ0B686N) がありました。

Reply:
修正しました